src/theme.ts

```diff
@@ -420,6 +420,8 @@ export default class MarkdownTheme extends Theme {
       contents
         .replace(/[\r\n]{3,}/g, '\n\n')
         .replace(/!spaces/g, '')
+        .replace('<', '&lt;')
```

Comment:
This should use a RegExp instead of string; otherwise it will only replace the first occurrence.

Reply:
Whoops, good catch. Fixed now. (Though I guess this isn't the desired solution.)